cmd/kubeadm/app/master/clusterroles.go

```diff
@@ -0,0 +1,87 @@
+package master
+
+import (
+	"fmt"
+
+	"k8s.io/kubernetes/pkg/api"
+	apiv1 "k8s.io/kubernetes/pkg/api/v1"
+	rbac "k8s.io/kubernetes/pkg/apis/rbac/v1alpha1"
+	"k8s.io/kubernetes/pkg/client/clientset_generated/clientset"
+)
+
+// CreateBootstrapRBACClusterRole creates the necessary ClusterRole for bootstrapping
+func CreateBootstrapRBACClusterRole(clientset *clientset.Clientset) error {
+	clusterRole := rbac.ClusterRole{
+		ObjectMeta: apiv1.ObjectMeta{Name: "kubeadm:kubelet-bootstrap"},
```

Comment:
I'm fine moving this into a bootstrap role. I can do that as a follow up PR and fix up the referenced name.